indico/modules/events/registration/templates/display/regform_list.html

```diff
@@ -1,8 +1,4 @@
-{% extends 'layout/event_page_base.html' %}
-
-{% block title %}
-    {%- trans %}Registration{% endtrans -%}
-{% endblock %}
+{% extends 'events/registration/display/_regform_base.html' %}
```

Comment:
True. It works because the subtitle block is overridden. But I think I'll inherit directly from the conference page template or add a `_event_base.html` template (which only inherits and sets the main title)